wgpu-hal/src/vulkan/device.rs

```diff
@@ -136,6 +136,15 @@ impl super::DeviceShared {
                         .pipeline_bind_point(vk::PipelineBindPoint::GRAPHICS)
                         .color_attachments(&color_refs)
                         .resolve_attachments(&resolve_refs);
+
+                    if self
+                        .workarounds
+                        .contains(super::Workarounds::EMPTY_RESOLVE_ATTACHMENT_LISTS)
+                        && vk_subpass.color_attachment_count == 0
+                    {
+                        vk_subpass.p_resolve_attachments = 0 as _;
```

Comment:
interesting choice, I'd expect `ptr::null()` instead

Reply:
Forgot it existed 😅